### PR TITLE
Borer infesting no longer requires input if there's only 1 target

### PR DESCRIPTION
**Pull request by Cyberboss**
Not sure if this is qualified under the freeze, but I've heard complaints about it

### CODE REVIEW

**Comment by KorPhaeron** [comment]
Balance

**Comment by Cyberboss** [comment]
It's not balance Kor, it's QoL. Read the diff!

**Comment by KorPhaeron** [comment]
I'm stupit

**Comment by KorPhaeron** [comment]
stop gaslighting me

**Comment by Cyberboss** [comment]
w-wew

**Comment by XDTM** [comment]
Shouldn't it do that by itself? I'm sure some kind of input already autopicks the only target if there's only one

**Comment by Cyberboss** [comment]
No, because of the `null|`

File: code/game/gamemodes/miniantags/borer/borer.dm [hunk 1]
```diff
@@ -336,7 +336,9 @@ var/total_borer_hosts_needed = 10
 		if(H!=src && Adjacent(H))
 			choices += H
 
-	var/mob/living/carbon/H = input(src,"Who do you wish to infest?") in null|choices
+	if(!choices.len)
+		return
+	var/mob/living/carbon/H = choices.len > 1 ? input(src,"Who do you wish to infest?") in null|choices : choices[1]
```
Comment (RemieRichards) [inline code/game/gamemodes/miniantags/borer/borer.dm]: if ```choices.len``` is 2 or more, we go to the input.
if it's 1, we go to ```choices[1]```
if it's 0... we go to ```choices[1]``` which doesn't exist, @CYBERBOSS  !!!

Comment (optimumtact) [inline code/game/gamemodes/miniantags/borer/borer.dm]: if choices len is 0 we are already returning earlier?

Comment (RemieRichards) [inline code/game/gamemodes/miniantags/borer/borer.dm]: oh right, duh

**Pull request merged**